Cargo.toml

```diff
@@ -28,6 +29,7 @@ encoding_rs = "0.8.28"
 encoding_rs_io = "0.1.7"
 flate2 = "1.0.22"
 indicatif = "0.16.2"
+indoc = "1.0"
```

Comment:
It's not a big deal, but do we need this just for that one string?

Reply:
You're right, I've now moved `indoc` back to dev-dependencies.